### PR TITLE
Immutable node setup

### PR DESCRIPTION
**Pull request by jp-imx**
Early access to our documentation.

* Exact docker image TBD.
* We are also troubleshooting some of the commands.

Sharing here to show progress and get early feedback.

### CODE REVIEW

File: src/pages/validator/external-chains/immutable.mdx [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+Partners can choose between a static IP allowlist or a WireGuard tunnel to establish their connections.
+
+The static IP allowlist ensures only a predetermined list of approved IP addresses can establish connections.
+
+The WireGuard tunnel provides partners with a secure and private conduit for their connections to the Immutable zkEVM,
+without requiring a static IP.
+
+<tabs>
+<tab-item title="Static IP">
+Using the Axelar CLI, create a transaction with a note that includes a `shasum` hash of your static IP.
```
Comment (canhtrinh) [inline src/pages/validator/external-chains/immutable.mdx:42]: "Using the Axelar CLI" -> "From your `vald` container"

File: src/pages/validator/external-chains/immutable.mdx [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+<tabs>
+<tab-item title="Testnet">
+```bash
+docker run \
+  -d \
+  --restart=always \
+  -v /opt/immutable-zkevm:/mnt/geth \
+  --name geth \
+  -p 8545:8545 \
+  geth \
```
Comment (canhtrinh) [inline src/pages/validator/external-chains/immutable.mdx:181]: small nit but there's a random newline here under testnet that doesn't appear under the mainnet tab (on line 196)

Comment (jp-imx) [reply src/pages/validator/external-chains/immutable.mdx:181]: Fixed to match

File: src/pages/validator/external-chains/immutable.mdx [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+<tab-item title="Mainnet">
+
+```bash
+curl https://rpc.immutable.com \
+  -X POST \
+  -H "Content-Type: application/json" \
+  --data '{"method":"eth_chainId","params":[],"id":1,"jsonrpc":"2.0"}'
+```
+
+The output should be:
```
Comment (milapsheth) [inline src/pages/validator/external-chains/immutable.mdx:333]: can you add the configure vald section like below?
https://github.com/axelarnetwork/axelar-docs/blob/main/src/pages/validator/external-chains/optimism.mdx#L174

**Review by milapsheth — APPROVED** [review]

File: src/pages/validator/external-chains/immutable.mdx [hunk 4]
```diff
@@ -336,3 +336,54 @@ The output should be:
...
+## Configure vald
+
+In order for `vald` to connect to your Optimism node, your `rpc_addr` should be exposed in
+vald's `config.toml`
+
+<tabs>
+  <tab-item title="Mainnet">
+  ```bash
+  [[axelar_bridge_evm]]
+  name = "optimism"
```
Comment (milapsheth) [inline src/pages/validator/external-chains/immutable.mdx:349]:
```suggestion
  name = "immutable"
```

File: src/pages/validator/external-chains/immutable.mdx [hunk 5]
```diff
@@ -336,3 +336,54 @@ The output should be:
...
+  rpc_addr = "http://IP:PORT"
+  start-with-bridge = true
+  ```
+  </tab-item>
+  <tab-item title="Testnet">
+  ```bash
+  [[axelar_bridge_evm]]
+  name = "optimism"
```
Comment (milapsheth) [inline src/pages/validator/external-chains/immutable.mdx:357]:
```suggestion
  name = "immutable"
```

File: src/pages/validator/external-chains/immutable.mdx [hunk 6]
```diff
@@ -336,3 +336,54 @@ The output should be:
...
+  start-with-bridge = true
+  ```
+  </tab-item>
+</tabs>
+
+### Common Node Operations
+
+First `cd` into the directory
+```bash
+cd $HOME/simple-optimism-node/
```
Comment (milapsheth) [inline src/pages/validator/external-chains/immutable.mdx:368]: update path

**Review by milapsheth — APPROVED** [review]

**Pull request merged**